test/conftest.py

```diff
@@ -228,3 +228,16 @@ def participants(db_session, a):
         db_session.add(p)
         db_session.flush()
     return ps
+
+# import dlgr was not working in the context of a test. The following
+# lines make sure it works.
+import sys
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
I don't love this, can't we `pip install -e .` the GridUniverse package to the `dlgr.griduniverse` package and the `dlgr` namespace registered?